src/styles/card.scss

```diff
@@ -0,0 +1,124 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: $card-width;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: calc($card-width - 40px);
+    height: 134px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: $font-size-base;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 40px 0 $small-margin;
+    color: #060b35;
+  }
+
+  &__code {
+    width: 100%;
+    display: flex;
+    justify-content: flex-start;
+    font-weight: 400;
+    color: $secondary-color;
+    margin-bottom: $medium-margin;
+
+    &-text {
+      font-size: $font-size-small;
+      line-height: 14px;
+      margin: 0;
+      color: $secondary-color;
+    }
+
+    &-number {
+      font-size: $font-size-small;
+      line-height: 14px;
+      margin: 0;
+      padding-inline-start: 2px;
+      color: $secondary-color;
+    }
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: center;
+    color: $main-accent-color;
+
+    &-text {
+      font-size: $font-size-small;
+      font-weight: 400;
+      line-height: 14px;
+      margin: 0;
+      color: $main-accent-color;
+    }
+
+    &-number {
+      font-family: Roboto, sans-serif;
+      font-size: $font-size-small;
+      color: $main-accent-color;
+      font-weight: 400;
+      line-height: 14px;
+      margin: 0 0 0 2px;
+    }
+  }
+
+  &__price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0 0 16px;
```

Comment:
Make sure that the `0` value in the margin shorthand property is necessary. If the left and right margins are not needed, you can use `margin-bottom: 16px;` instead for better clarity.